rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -535,7 +535,7 @@ class spu_thread : public cpu_thread
 
 	// Reservation Data
 	u64 rtime = 0;
-	std::array<u128, 8> rdata{};
```

Comment:
What is the purpose of this change?

Reply:
It's not aligned badly in the first place. We cannot yet guarantee portability compiler support for alignments more than 16.

Reply:
fair enough.